### PR TITLE
OpenSearch Exporter initial configuration 

### PR DESCRIPTION
**Pull request by MitchellGale**
OpenSearch Exporter initial PR for set-up.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -66,6 +66,7 @@ exporter/logzioexporter/                                 @open-telemetry/collect
 exporter/lokiexporter/                                   @open-telemetry/collector-contrib-approvers @gramidt @gouthamve @jpkrohling @kovrus @mar4uk
 exporter/mezmoexporter/                                  @open-telemetry/collector-contrib-approvers @dashpole @billmeyer @gjanco
 exporter/opencensusexporter/                             @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
+exporter/opensearchexporter/                             @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers @MaxKsyunz @MitchellGale
```
Comment (MitchellGale) [inline .github/CODEOWNERS:69]: **INFO:** Errors are from all the code owners not having write access

File: cmd/otelcontribcol/builder-config.yaml [hunk 2]
```diff
@@ -66,6 +103,7 @@ exporters:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.80.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+    - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```
Comment (matthewryanwells) [inline cmd/otelcontribcol/builder-config.yaml]:
```suggestion
  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```

Looks like this line is indented further than the other lines around it

File: cmd/otelcontribcol/go.mod [hunk 3]
```diff
@@ -40,6 +40,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```
Comment (matthewryanwells) [inline cmd/otelcontribcol/go.mod]: Spacing on this line is off compared to the other lines

File: cmd/otelcontribcol/builder-config.yaml (hunk 2 above)
Comment (MitchellGale) [reply cmd/otelcontribcol/builder-config.yaml]: Thanks!

File: cmd/otelcontribcol/go.mod (hunk 3 above)
Comment (MitchellGale) [reply cmd/otelcontribcol/go.mod]: Thanks!

File: cmd/oteltestbedcol/go.mod [hunk 4]
```diff
@@ -8,6 +8,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbonexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```
Comment (matthewryanwells) [inline cmd/oteltestbedcol/go.mod]: Spacing

**Review by matthewryanwells — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Review by GumpacG — APPROVED** [review]

**Pull request closed without merging**